partner_contact_nutrition_allergen/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    nutrition_allergen_ids = fields.Many2many(comodel_name='product.product',
+                                              string='Allergens',
+                                              domain="[('type', '=',"
```

Comment:
Would it not be advisable to have an 'is_allergen' boolean on product.template? This would make selection much more precise, then a selection in all stockable products.